### DISCUSSION TITLE
automation to set charge time

### DISCUSSION THREAD

**Discussion by motclem** [wills106/homeassistant-solax-modbus#23] [Q&A]
iit I have the current price of electricity(agile octopus) as a data point in my ha I would like to be able to set my  SolaX battery to charge when the price is below a defined threshold is this currently possible? if so any pointers I was thinking a trigger of when price is below x >> set battery tp charge or is this too simplistic ?

thanks,

tom

**Comment by wills106** [comment]
#27 Starting to add this in, but it's limited functionality at the moment and limited to 15min chunks.
